mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulFollowingScreen.kt

```diff
@@ -42,27 +48,35 @@ fun StatefulFollowingScreen(
           .map { ProfileAdapter(it) }
 
   val searchFieldInteraction = remember { MutableInteractionSource() }
-  val focused by searchFieldInteraction.collectIsFocusedAsState()
-  val mode = if (focused) Searching else Following
+  val focused = searchFieldInteraction.collectIsFocusedAsState()
+  val mode = if (focused.value) Searching else Following
+  val scope = rememberCoroutineScope()
 
   SocialScreen(
-      SnapshotSocialScreenState(following, input, searchResults, mode, searchFieldInteraction),
-      modifier)
+      SnapshotSocialScreenState(
+          following, input, searchResults, mode, searchFieldInteraction, socialFacade, scope),
+      modifier.fillMaxSize())
```

Comment:
It is not the responsibility of this composable to apply `Modifier.fillMaxSize()`.